packages/ui/src/pages/About.tsx

```diff
@@ -18,13 +19,13 @@ const About = ({ className }: Props) => {
           <h2>Open</h2>
           <p>
             Multix is open-source from the get-go. The back-end and front-end code can be{' '}
-            <a
+            <Link
```

Comment:
These look off. I'll revert the use of `Link` instead of `a`.
![image](https://github.com/ChainSafe/Multix/assets/33178835/15db2329-2549-4ee3-aefa-126aa6195ae0)